.github/workflows/review_apps_on_pr_change.yml

```
@@ -64,10 +64,12 @@ jobs:
             -no-color \
             -auto-approve
 
-          echo "REVIEW_APP_URL=$(terraform output -raw review_app_url)" >> "$GITHUB_OUTPUT"
-          echo "ADMIN_APP_URL=$(terraform output -raw admin_app_url)" >> "$GITHUB_OUTPUT"
-          echo "ECS_CLUSTER_ID=$(terraform output -raw review_app_ecs_cluster_id)" >> "$GITHUB_OUTPUT"
-          echo "ECS_SERVICE_NAME=$(terraform output -raw review_app_ecs_service_name)" >> "$GITHUB_OUTPUT"
+          {
+            echo "REVIEW_APP_URL=$(terraform output -raw review_app_url)"
+                 "ADMIN_APP_URL=$(terraform output -raw admin_app_url)"
+                 "ECS_CLUSTER_ID=$(terraform output -raw review_app_ecs_cluster_id)"
+                 "ECS_SERVICE_NAME=$(terraform output -raw review_app_ecs_service_name)"
+          } >> "$GITHUB_OUTPUT"
```

I think this linter change broke the review apps action... bash is trying to run `ADMIN_APP_URL=whatever` as a ccommand. Some line continuations might help?

```suggestion
          {
            echo "REVIEW_APP_URL=$(terraform output -raw review_app_url)" \
                 "ADMIN_APP_URL=$(terraform output -raw admin_app_url)" \
                 "ECS_CLUSTER_ID=$(terraform output -raw review_app_ecs_cluster_id)" \
                 "ECS_SERVICE_NAME=$(terraform output -raw review_app_ecs_service_name)"
          } >> "$GITHUB_OUTPUT"
```